src/Sixel.Supported.cs

```diff
@@ -0,0 +1,133 @@
+using SixLabors.ImageSharp;
+
+namespace SixPix;
+
+public partial class Sixel
+{
+    private static Size? CellSize = null;
+    private static Size? WindowPixelSize = null;
+    private static Size? WindowCharSize = null;
+
+    private const string CSI_DEVICE_ATTRIBUTES = "[c";
+    private const string CSI_CELL_SIZE = "[16t";
+    private const string CSI_WINDOW_PIXSIZE = "[14t";
+    private const string CSI_WINDOW_CHARSIZE = "[18t";
+
+    /// <summary>
+    /// Check whether current terminal support Sixel
+    /// <returns>bool</returns>
+    /// </summary>
+    public static bool IsSupported()
+    {
+
+        // Should get something like: ^[[?61;4;6;7;14;21;22;23;24;28;32;42c
+        // The "4" indicates Sixel support
+        var response = GetCtrlSeqResponse(CSI_DEVICE_ATTRIBUTES);
+
+        if (response.Contains(";4;"))
+            return true;
```

Comment:
FYI, the `4` could also be the last parameter in the response, so you should also check `";4c"`. For example, the DA response from Konsole is `^[[?62;1;4c`.

Reply:
Ah, good catch.

Reply:
I stole this from https://github.com/trackd/Sixel so I'll submit a PR there.